packages/react/src/NavList/NavList.stories.tsx

```diff
@@ -246,4 +247,26 @@ export const WithGroup = () => (
   </PageLayout>
 )
 
+export const WithTrailingAction = () => {
```

Comment:
Non-blocking: A story showing this feature [within sub items](https://primer.style/react/storybook/?path=/story/components-navlist--with-nested-sub-items) could be cool!

Reply:
Done!

See [Storybook: WithTrailingActionInSubItem](https://primer-cb00a13080-13348165.drafts.github.io/storybook/?path=/story/components-navlist--with-trailing-action-in-sub-item)